osu.Game/Skinning/LegacySkin.cs

```diff
@@ -257,6 +257,24 @@ private IBindable<TValue> lookupForMania<TValue>(LegacyManiaSkinConfigurationLoo
                 case LegacyManiaSkinConfigurationLookups.RightLineWidth:
                     Debug.Assert(maniaLookup.TargetColumn != null);
                     return SkinUtils.As<TValue>(new Bindable<float>(existing.ColumnLineWidth[maniaLookup.TargetColumn.Value + 1]));
+
+                case LegacyManiaSkinConfigurationLookups.Hit0:
+                    return SkinUtils.As<TValue>(getManiaImage(existing, "Hit0"));
```

Comment:
why is this done this way, not in a similar way to existing `NoteImage` / `KeyImage`?

Reply:
What do you mean? It is done similar to those.

Reply:
Yeah, realised they are all using column as the variable. Not sure it's worth making a custom lookup type just for hit results, but I guess I am hoping to not have separate implementations for each hit value (taiko's `LegacySkinTransformer` is probably the best example)

Reply:
This is a step above taiko's transformer, because taiko doesn't have the ability to skin individual images via skin.ini in the first place.
One potential code quality improvement is creating a different type to pass to the legacy skin, like `LegacyManiaSkinConfigurationHitLookup`, which could take a string and probably centralise things into the transformer. But it's legacy so /shrug from me.

Reply:
Fair. Are you okay with the following?

```csharp

                case LegacyManiaSkinConfigurationLookups.Hit0:
                case LegacyManiaSkinConfigurationLookups.Hit50:
                case LegacyManiaSkinConfigurationLookups.Hit100:
                case LegacyManiaSkinConfigurationLookups.Hit200:
                case LegacyManiaSkinConfigurationLookups.Hit300:
                case LegacyManiaSkinConfigurationLookups.Hit300g:
                    return SkinUtils.As<TValue>(getManiaImage(existing, maniaLookup.Lookup.ToString()));
```

Reply:
Looks fine